apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Logging/GoogleLogger.cs

```diff
@@ -230,13 +232,48 @@ public Uri GetGcpConsoleLogsUrl()
             }.Uri;
         }
 
+        /// <summary>
+        /// For diagnostic purposes. Builds and returns the URL where the entries logged by
+        /// this <see cref="GoogleLogger"/> can be seen on the Google Cloud Logs Explorer.
+        /// </summary>
+        public Uri GetLogsExplorerLogsUrl()
```

Comment:
Possibly extract common method, given that these only differ (as far as I can see) by _fullLogName or _logName? (Maybe there's more that I've missed.)